modules/serverless-pingsource-ODC.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * serverless/event_workflows/serverless-pingsource.adoc
+
+[id="serverless-pingsource-ODC_{context}"]
+= Creating a PingSource using {product-title} web console
+
+The following sections describe how to create and verify a basic PingSource from the {product-title} web console.
+
+.Prerequisites
+
+* You have Knative Serving and Eventing installed.
+
+.Procedure
+
+. To verify that the PingSource is working, create a simple Knative
+service that dumps incoming messages to the service's logs.
+.. In the *Developer* perspective, navigate to *+Add* -> *YAML*.
+.. Copy the example YAML:
++
```

Comment:
can as well create `knative service` via form Experience
goto Add -> conatinerImage -> provide image -> select ksvc in radio option -> click create

![image](https://user-images.githubusercontent.com/5129024/95093424-66478380-0746-11eb-9ab0-cfb083cf1da0.png)